VATSpy.dat

```diff
@@ -1,4 +1,4 @@
-[Countries]
+L[Countries]

```

Comment:
Typo?

Reply:
both LOVV_N and LOVV_S should be removed, so that only LOVV sector remain.
what you mean verified your github account? thats not my first change in the gitlab, cause I inserted this 2 sectors at 08. May 2020

Reply:
We've described a new process to verify your account by sending an email to @prryplatypus. When you're on his list we can approve.